auth-web/src/components/auth/manage-business/CreateNewBusinessHelp.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <section class="help-content">
+    <header class="d-flex justify-center mt-4">
+      <p class="title-text mt-4">{{ helpContent.title }}</p>
+    </header>
+    <div class="help-text mt-4">
+      <p>{{ helpContent.intro }}</p>
+      <ol>
+        <li class="mt-12" v-for="(step, index) in helpContent.steps" :key="index">
+          <strong>{{ step.title }}</strong>
+          <ul class="mt-8">
+            <li v-for="(item, i) in step.items" :key="i">
+              <p v-if="typeof item === 'string'">{{ item }}</p>
+              <p v-else v-html="item.content"></p>
+            </li>
+          </ul>
+        </li>
+      </ol>
+    </div>
+  </section>
+</template>
+
+<script>
+export default {
+  data () {
+    return {
+      helpContent: {
+        'title': 'Help with Starting a Business',
+        'intro': 'Start a named or numbered business in B.C. by following these steps:',
+        'steps': [
+          {
+            'title': 'Decide on a Business Type',
+            'items': [
+              {
+                'content': "Decide which business structure is the most appropriate for you. A few options are: a sole proprietorship, partnership, or corporation. Each structure has different legal and financial implications. <a href='https://entity-selection-prod.apps.silver.devops.gov.bc.ca/'>Use the Business Structures Wizard to help you decide.</a>",
```

Comment:
Still missing icon after link. Should look like uxpin:
![image](https://github.com/bcgov/sbc-auth/assets/10002889/cb325c5a-a1db-48ae-b421-24171a44fd0a)